script/github-actions/get-changed-apps.unit.spec.js

```diff
@@ -92,6 +92,14 @@ describe('getChangedAppsString', () => {
     expect(appString).to.equal('src/applications/app1,src/applications/app2');
   });
 
+  it('should return a comma-delimited string of app URLs when the url output type is specified', () => {
```

Comment:
I think these unit tests are really paying off as we extend the functionality of this script

Reply:
Totally, they've been super helpful for validating the functionality of the script. As we add support for grouped apps, they will be even more valuable